### PR TITLE
issue-474: Added input label for actions and fixed console errors.

### PR DESCRIPTION
**Pull request by anandtoshniwal**
#474 

## Screenshot / UI changes

screen shot 2018-09-14 at 1 53 36 pm
screen shot 2018-09-14 at 1 54 44 pm

## Testing instructions

Go to taxonomy and taxonomy term overview page and check for the actions input label
Go to taxonomy term overview page and check the console for errors.

### CODE REVIEW

File: src/components/05_pages/TaxonomyTermsOverview/TaxonomyTermsOverview.js [hunk 1]
```diff
@@ -74,30 +86,32 @@ export default class TaxonomyTermsOverview extends React.Component {
...
-            </TableHead>
-            <TableBody>
-              {taxonomyTerms &&
-                taxonomyTerms.map(term => (
-                  <TableRow key={term.attributes.uuid}>
-                    <TableCell>{term.attributes.name}</TableCell>
-                    <TableCell>
-                      {this.termOperations(term.attributes.vid)}
-                    </TableCell>
+          <div>
```
[the remark this answers is not in the document]
Comment (anandtoshniwal) [reply src/components/05_pages/TaxonomyTermsOverview/TaxonomyTermsOverview.js]: yes, we need the div.. before adding div tried to put condition with <Fragment> but it didn't work. It crashes the page and returns the error.

[the remark this answers is not in the document]
Comment (dawehner) [reply src/components/05_pages/TaxonomyTermsOverview/TaxonomyTermsOverview.js]: What happened exactly when you used ```<Fragment>```?

[the remark this answers is not in the document]
Comment (anandtoshniwal) [reply src/components/05_pages/TaxonomyTermsOverview/TaxonomyTermsOverview.js]: @dawehner 
please check the following screenshot
screen shot 2018-09-14 at 5 35 27 pm

[the remark this answers is not in the document]
Comment (anandtoshniwal) [reply src/components/05_pages/TaxonomyTermsOverview/TaxonomyTermsOverview.js]: I guess it needs wrapper around it.. which why I added div to it so fragment can be used as wrapper.

**Pull request merged**